plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationProviderImpl.kt

```diff
@@ -63,9 +65,16 @@ internal class LocationProviderImpl(context: Context) :
     locationConsumers.forEach { consumer ->
       consumer.onLocationUpdated(Point.fromLngLat(location.longitude, location.latitude))
       consumer.onBearingUpdated(location.bearing.toDouble())
+      if (accuracyRadiusSettings.showAccuracyRing && consumer is LocationConsumer2) {
```

Comment:
IMO location provider shouldn't know about settings of the display logic of locations. Then here we could only check `if (consumer is LocationConsumer2) {` and accuracyRadiusSettings can be checked within  location plugin or location puck

Reply:
Right, we shouldn't handle it in location provider. This is handled in `accuracyRadiusAnimator`